solana/bridge/program/src/lib.rs

```diff
@@ -8,7 +8,6 @@ use solitaire::*;
 pub const MAX_LEN_GUARDIAN_KEYS: usize = 19;
 pub const CHAIN_ID_SOLANA: u16 = 1;
 
-#[cfg(feature = "no-entrypoint")]
```

Comment:
I don't think we should remove this, we don't want the instruction builders built into the on-chain program. Maybe we need to add a different feature flag to handle whatever this was needed for?

Reply:
The tests need the builder functions.  I ran into some other issues getting the tests working with the "no-entrypoint" feature enabled so I added a separate feature for the instructions module only.